src/app/app.config.ts

```diff
@@ -0,0 +1,55 @@
+import { ApplicationConfig, InjectionToken, importProvidersFrom } from '@angular/core';
+import { provideRouter } from '@angular/router';
+
```

Comment:
Let's use the following rules for imports:
1. Separate all import into two blocks divided by an empty line
2. First block is for angular imports and third-party libraries
3. Second block is for importing your own services/constants/etc.

This approach is not only prettier but also makes code more readable